### PR TITLE
Added Media Type

### PR DESCRIPTION
**Pull request by cholmes**
Added [application/vnd.apache.parquet](https://www.iana.org/assignments/media-types/application/vnd.apache.parquet) as the media type, as discussed in #115

Like #212 I went with no 'justification' as to why we don't have a 'geo' type, but could add that in.

This is likely not the right organizational structure, but working to get the PR in, and then we can revisit where it goes.

### CODE REVIEW

File: format-specs/geoparquet.md [hunk 1]
```diff
@@ -289,3 +289,7 @@ The CRS is likely equivalent to OGC:CRS84 for a GeoParquet file if the `id` elem
 It is reasonable for implementations to require that one of the above `id` elements are present and skip further tests to determine if the CRS is functionally equivalent with OGC:CRS84.
 
 Note: EPSG:4326 and OGC:CRS84 are equivalent with respect to this specification because this specification specifically overrides the coordinate axis order in the `crs` to be longitude-latitude.
+
+## Media Type
+
+A GeoParquet file ([RFC 8259](https://tools.ietf.org/html/rfc8259)) SHOULD use [`application/vnd.apache.parquet`](https://www.iana.org/assignments/media-types/application/vnd.apache.parquet) as the [Media Type](https://en.wikipedia.org/wiki/Media_type) (previously known as the MIME Type).
```
Comment (m-mohr) [inline format-specs/geoparquet.md:295]: Why don't we go for a MUST here?

The RFC link seems to be out of place?

Comment (cholmes) [inline format-specs/geoparquet.md:295]:
```suggestion
A GeoParquet file SHOULD use [`application/vnd.apache.parquet`](https://www.iana.org/assignments/media-types/application/vnd.apache.parquet) as the [Media Type](https://en.wikipedia.org/wiki/Media_type) (previously known as the MIME Type).
```

Comment (m-mohr) [reply format-specs/geoparquet.md]: @cholmes You edited my post 😂 Happy that it's not just me who does this by accident ;-)

File: format-specs/geoparquet.md [hunk 2]
```diff
@@ -289,3 +289,7 @@ The CRS is likely equivalent to OGC:CRS84 for a GeoParquet file if the `id` elem
 It is reasonable for implementations to require that one of the above `id` elements are present and skip further tests to determine if the CRS is functionally equivalent with OGC:CRS84.
 
 Note: EPSG:4326 and OGC:CRS84 are equivalent with respect to this specification because this specification specifically overrides the coordinate axis order in the `crs` to be longitude-latitude.
+
+## Media Type
+
+A GeoParquet file SHOULD use [`application/vnd.apache.parquet`](https://www.iana.org/assignments/media-types/application/vnd.apache.parquet) as the [Media Type](https://en.wikipedia.org/wiki/Media_type) (previously known as the MIME Type).
```
Comment (m-mohr) [inline format-specs/geoparquet.md:295]:
```suggestion
If a [media type](https://en.wikipedia.org/wiki/Media_type) (formerly: MIME type) is used, a GeoParquet file MUST use [application/vnd.apache.parquet](https://www.iana.org/assignments/media-types/application/vnd.apache.parquet) as media type.
```

File: format-specs/geoparquet.md (hunk 1 above)
Comment (cholmes) [reply format-specs/geoparquet.md]: Double fail! I'll just leave this be then. New proposal looks good, I'll add it.

File: format-specs/geoparquet.md [hunk 3]
```diff
@@ -289,3 +289,7 @@ The CRS is likely equivalent to OGC:CRS84 for a GeoParquet file if the `id` elem
 It is reasonable for implementations to require that one of the above `id` elements are present and skip further tests to determine if the CRS is functionally equivalent with OGC:CRS84.
 
 Note: EPSG:4326 and OGC:CRS84 are equivalent with respect to this specification because this specification specifically overrides the coordinate axis order in the `crs` to be longitude-latitude.
+
+## Media Type
+
+If a [media type](https://en.wikipedia.org/wiki/Media_type) (formerly: MIME type) is used, a GeoParquet file MUST use [application/vnd.apache.parquet](https://www.iana.org/assignments/media-types/application/vnd.apache.parquet) as media type.
```
Comment (cholmes) [inline format-specs/geoparquet.md:295]:
```suggestion
If a [media type](https://en.wikipedia.org/wiki/Media_type) (formerly: MIME type) is used, a GeoParquet file MUST use [application/vnd.apache.parquet](https://www.iana.org/assignments/media-types/application/vnd.apache.parquet) as the media type.
```

**Review by m-mohr — APPROVED** [review]

**Review by kylebarron — APPROVED** [review]

**Review by jorisvandenbossche — APPROVED** [review]

**Comment by kylebarron** [comment]
CI is currently broken from https://github.com/opengeospatial/geoparquet/pull/214

**Comment by m-mohr** [comment]
CI has been fixed

**Pull request merged**